vhdbuilder/packer/install-dependencies.sh

```diff
@@ -543,7 +543,7 @@ for ADDON_IMAGE in ${ADDON_IMAGES}; do
 done
 
 AZUREDISK_CSI_VERSIONS="
-0.7.0
```

Comment:
if 0.7.0  is being used by exsiting cluster shall we keep it until 0.9.0 is fully in effect (as part of the ccp chart upgrade )?

Reply:
Yes, we should keep 0.7.0 version. Updated PR.